xml/System/IntPtr.xml

```diff
@@ -632,7 +632,7 @@
         <ReturnType>System.IntPtr</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Represents the largest possible value of <see cref="T:System.IntPtr" />.</summary>
         <value>To be added.</value>
```

Comment:
Missing the value description here. How does this sound?:
```suggestion
        <value>A number that represents the largest possible value of <see cref="T:System.IntPtr" />.</value>
```
If you prefer something different, feel free to edit the suggestion and the merge it.

Reply:
This was copied from the docs used for Min/MaxValue on the other primitive types.